.circleci/main.yml

```diff
@@ -1163,7 +1163,60 @@ jobs:
                 echo "regular release."
                 gh release create --draft ${CIRCLE_TAG} --title ${CIRCLE_TAG:1} ./dist-pach/pachctl/pachctl_${CIRCLE_TAG:1}* ./dist-pach/mount-server/mount-server_${CIRCLE_TAG:1}* --generate-notes
             fi
+  console-release-draft:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - gh/setup:
+          version: 2.13.0
+      - run:
+          name: create github console release draft
+          command: |
+            if [[ $CIRCLE_TAG == *"-"* ]];
+              then
+                  echo "git tag has - assuming its a prerelease."
+                  parentBranch=master
+              else
+                  echo "regular release."
+                  parentBranch=${CIRCLE_TAG:1}
+              fi
+            git clone https://github.com/pachyderm/haberdashery.git haberdashery
+            git checkout -b "release-${CIRCLE_TAG:1}"
+            tmp=$(mktemp)
+            jq --arg val "${CIRCLE_TAG:1}-${CIRCLE_SHA1}" '.pachyderm = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_SHA1}" '.pachReleaseCommit = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_TAG:1}" '.pachctl = $val' version.json > "$tmp" && mv "$tmp" version.json
+            git commit -m "bump pachyderm version"
+            git push
+            PR=$(gh pr create --base $parentBranch --title "bump pach version for release ${CIRCLE_TAG:1}" --repo pachyderm/haberdashery)
+            gh pr merge ${PR} --auto --repo pachyderm/haberdashery
+            gh pr checks ${PR} --10 --watch --repo pachyderm/haberdashery
```

Comment:
this is pretty slick! What is the `--10`? I was hoping to find a timeout setting in the docs, but I couldn't find one.

Reply:
I think I ate the `-i` flag. yea its the only glue I could find https://cli.github.com/manual/gh_pr_checks